stl/debugger/STL.natvis

```diff
@@ -1112,8 +1112,8 @@ SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
   <Type Name="std::basic_string_view&lt;*,*&gt;">
     <Intrinsic Name="size" Expression="_Mysize" />
     <Intrinsic Name="data" Expression="_Mydata" />
-    <DisplayString>{_Mydata,[_Mysize]}</DisplayString>
-    <StringView>_Mydata,[_Mysize]</StringView>
+    <DisplayString>{_Mydata,[_Mysize]na}</DisplayString>
+    <StringView>_Mydata,[_Mysize]na</StringView>

```

Comment:
Thank you for noticing and fixing this! `wstring_view` et al. and `u32string_view` need similar fixes:

https://github.com/microsoft/STL/blob/89ca073a866d24db42b3f4c5e406be4b28a148f4/stl/debugger/STL.natvis#L1004-L1006

<img width="459" alt="string_view" src="https://github.com/user-attachments/assets/78b23258-9906-48c2-8ef7-7e4c23c80e35">

```cpp
#include <iostream>
#include <string_view>
using namespace std;

int main() {
    string_view sv{"cute fluffy kittens"};
    wstring_view wsv{L"long long cats"};
    u32string_view u32sv{U"cats rule the world"};
    cout << "Bye!" << endl;
}
```

Reply:
my fix should take care of all these, as they are typedefs of `basic_string_view`:
![image](https://github.com/user-attachments/assets/f13dc70e-71ff-4a81-84d6-7afb700e4b8d)

Note, to see it all fixed you'll need to patch manually your `Common7\Packages\Debugger\Visualizers\stl.natvis`

Reply:
if I undo my fix, this is how it looks like:
![image](https://github.com/user-attachments/assets/ad74dfa2-83a5-4f55-947a-fad89c1ea342)

Reply:
it seems that `na` is ok for all of these.

Reply:
Interesting! I confirm that `na` works even if the `wstring_view` and `u32string_view` contain a cat emoji with `\U0001F408`. Good enough for me.

Reply:
> Note, to see it all fixed you'll need to patch manually your `Common7\Packages\Debugger\Visualizers\stl.natvis` inside VS install dir.

Or add a new NATVIS file to your project with the contents of `STL.natvis` from the PR:
![image](https://github.com/user-attachments/assets/764a274e-51f0-4c42-a92f-46eab8b13d8f)
(Ignore the pre-existing issue with embedded null characters in my screenshot.)